app/components/form/title_component.rb

```diff
@@ -1,11 +1,12 @@
 # frozen_string_literal: true
 
 class Form::TitleComponent < ViewComponent::Base
-  def initialize(title:, subtitle:, step: nil, total_steps: nil, notes: nil, autosave: false)
+  def initialize(title:, subtitle:, step: nil, total_steps: nil, notes: nil, autosave: false, navigable: nil)
```

Comment:
new optional param for navigating through form via buttons near progress bar

Reply:
This ends up with suuuuper long call to render method. How about we just use a slot? https://viewcomponent.org/guide/slots.html#component-slots